admin/framework/composer.json

```diff
@@ -17,7 +17,7 @@
     "require-dev": {
         "codeigniter/coding-standard": "^1.1",
         "fakerphp/faker": "^1.9",
-        "friendsofphp/php-cs-fixer": "3.6.*",
+        "friendsofphp/php-cs-fixer": "~3.10.0",
```

Comment:
Why `~3.10.0`? `^3.10` seems fine.

Reply:
The tilde is the same with the `*`. It limits version to the patch

Reply:
```console
$ git diff
diff --git a/composer.json b/composer.json
index 080fc5974..e471a6452 100644
--- a/composer.json
+++ b/composer.json
@@ -17,7 +17,7 @@
     "require-dev": {
         "codeigniter/coding-standard": "^1.1",
         "fakerphp/faker": "^1.9",
-        "friendsofphp/php-cs-fixer": "3.6.*",
+        "friendsofphp/php-cs-fixer": "~3.9.0",
         "mikey179/vfsstream": "^1.6",
         "nexusphp/cs-config": "^3.3",
         "nexusphp/tachycardia": "^1.0",
```
```console
$ composer update
Loading composer repositories with package information
Updating dependencies
Lock file operations: 0 installs, 2 updates, 1 removal
  - Removing php-cs-fixer/diff (v2.0.2)
  - Upgrading friendsofphp/php-cs-fixer (v3.6.0 => v3.9.6)
  - Upgrading nexusphp/cs-config (v3.4.0 => v3.5.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 0 installs, 2 updates, 1 removal
  - Removing php-cs-fixer/diff (v2.0.2)
  - Upgrading friendsofphp/php-cs-fixer (v3.6.0 => v3.9.6): Extracting archive
  - Upgrading nexusphp/cs-config (v3.4.0 => v3.5.0): Extracting archive
Generating optimized autoload files
53 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
infection/extension-installer: No extensions found
> CodeIgniter\ComposerScripts::postUpdate
> bash -c "if [ -f admin/setup.sh ]; then bash admin/setup.sh; fi"
No security vulnerability advisories found
```

Reply:
It prevents us from going to `3.11.x` which I believe is our desired behavior. 

https://getcomposer.org/doc/articles/versions.md#tilde-version-range-